aries-site/src/examples/templates/filtering/FilteringCards/FilteringCards.js

```diff
@@ -1,101 +1,73 @@
 import { useContext } from 'react';
-import PropTypes from 'prop-types';
 import {
   Box,
   Card,
+  Cards,
   CardBody,
-  Grid,
+  Data,
+  DataFilters,
+  DataSearch,
+  DataSummary,
   Heading,
+  Page,
+  PageContent,
   ResponsiveContext,
   Text,
+  Toolbar,
 } from 'grommet';
 
-import {
-  FilterControls,
-  FiltersProvider,
-  useFilters,
-} from '../../FilterControls';
 import { users } from './mockData';
 
-export const FilteringCards = ({ containerRef }) => {
-  // containerRef is for demonstration purposes on this site. Most
-  // implementations should likely remove.
-
-  const size = useContext(ResponsiveContext);
-
-  // Define which attributes should be made available for the user
-  // to filter upon
-  const filtersConfig = [
-    { property: 'role', label: 'Role', filterType: 'CheckBoxGroup' },
-    { property: 'status', label: 'Status', filterType: 'CheckBoxGroup' },
-    {
-      property: 'location',
-      label: 'Location',
-      filterType: 'CheckBoxGroup',
-    },
-    {
-      property: 'hoursAvailable',
-      label: 'Remaining hours available',
-      filterType: 'RangeSelector',
-      inputProps: {
-        min: 0,
-        max: 40,
-        valueRange: '0 - 40 hours',
-      },
-    },
-    { property: 'name', label: 'Name', filterType: 'CheckBoxGroup' },
-  ];
-
-  // Customize layer properties. Any Grommet Layer props, plus Box props
-  // for the Layer's container and contents.
-  const layerProps = {
-    // containerRef is for demonstration purposes on this site. Most
-    // implementations should likely remove.
-    target: containerRef && containerRef.current,
-  };
+// Define Data properties
+const properties = {
+  role: { label: 'Role' },
+  status: { label: 'Status' },
+  location: { label: 'Location' },
+  hoursAvailable: {
+    label: 'Remaining hours available',
+    range: { min: 0, max: 40 },
+  },
+  name: { label: 'Name' },
+};
 
-  return (
-    <Box
-      background="background"
-      pad={!['xsmall', 'small'].includes(size) ? 'large' : 'medium'}
-      gap="medium"
-    >
-      <Heading level={2} margin="none">
-        Users
-      </Heading>
-      <FiltersProvider>
-        <Box gap="medium">
-          <FilterControls
+export const FilteringCards = () => (
+  <Page fill>
```

Comment:
I see, this happened because of a slight shortcut we took with regards to removing the background instead of adding the padding. I'll make sure the fullscreen also respects padding defined by Page.

Reply:
After discussing with Taylor, we decided to keep this as a separate issue regarding the Example container. The goal in mind is that the user should not see any code that manually adds padding that the Example container is supposed to provide. By creating an issue for the Example container, this would prevent having to add padding on every single local example.